src/Designer/frontend/dashboard/pages/OrgContentLibraryPage/OrgContentLibraryPage.test.tsx

```diff
@@ -307,33 +306,46 @@ describe('OrgContentLibraryPage', () => {
     renderOrgContentLibraryWithData({ featureFlags: [FeatureFlag.NewCodeLists] });
     const pagesConfig = retrievePagesConfig();
     const { codeLists } = pagesConfig.codeLists.props;
-    expect(codeLists).toHaveLength(codeListsNewResponse.codeListWrappers.length);
+    expect(codeLists).toHaveLength(sharedResourcesResponse.files.length);
   });
 
-  it('Calls updateOrgCodeLists with correct data when code list saving is triggered on the new code list page', async () => {
-    const updateOrgCodeLists = jest.fn();
+  it('Calls updateSharedResources with correct data when code list saving is triggered on the new code list page', async () => {
+    const updateSharedResources = jest.fn();
     renderOrgContentLibraryWithData({
       featureFlags: [FeatureFlag.NewCodeLists],
-      queries: { updateOrgCodeLists },
+      queries: { updateSharedResources },
     });
 
     const newCodeLists: LibraryCodeListData[] = [
       { name: 'list-1', codes: [{ value: '8', label: { nb: 'Åtte' } }] },
       { name: 'list-2', codes: [{ value: '9', label: { en: 'Nine' } }] },
     ];
     retrievePagesConfig().codeLists.props.onSave(newCodeLists);
-    await waitFor(expect(updateOrgCodeLists).toHaveBeenCalled);
-
-    const expectedPayload: UpdateOrgCodeListsPayload = {
-      baseCommitSha: codeListsNewResponse.commitSha,
-      codeListWrappers: expect.arrayContaining([
-        { title: newCodeLists[0].name, codeList: { codes: newCodeLists[0].codes } },
-        { title: newCodeLists[1].name, codeList: { codes: newCodeLists[1].codes } },
-      ]),
+    await waitFor(expect(updateSharedResources).toHaveBeenCalled);
+
+    expect(updateSharedResources).toHaveBeenCalledTimes(1);
+    expect(updateSharedResources).toHaveBeenCalledWith(orgName, {
+      files: [
+        {
+          path: 'CodeLists/list-1.json',
+          content: JSON.stringify({ codes: newCodeLists[0].codes }, null, 2),
+        },
+        {
+          path: 'CodeLists/list-2.json',
+          content: JSON.stringify({ codes: newCodeLists[1].codes }, null, 2),
+        },
+        {
+          path: 'CodeLists/animals.json',
+          content: '',
+        },
+        {
+          path: 'CodeLists/vehicles.json',
+          content: '',
+        },
+      ],
+      baseCommitSha: sharedResourcesResponse.commitSha,
       commitMessage: textMock('org_content_library.code_lists.commit_message_default'),
-    };
-    expect(updateOrgCodeLists).toHaveBeenCalledTimes(1);
-    expect(updateOrgCodeLists).toHaveBeenCalledWith(orgName, expectedPayload);
+    });
```

Comment:
Her kan du bruke `satisfies` for å få med typesikring:
```suggestion
    } satisfies UpdateSharedResourcesRequest);
```
Eventuelt bare plassere dataen i en typet variabel, slik den var.